widgets/common/src/BlueprintActions.js

```diff
@@ -58,6 +58,7 @@ export default class BlueprintActions {
         deploymentId,
         inputs,
         visibility,
+        labels,
```

Comment:
`labels` are not passed in in `ConfirmationStep`. I would be able to understand it easier if there was a default value, rather than relying on this function handling both `undefined` and a valid array :slightly_smiling_face: 

```suggestion
        labels = [],
```

Reply:
Done